builder/vmware/iso/builder.go

```diff
@@ -136,7 +136,7 @@ func (b *Builder) Run(ctx context.Context, ui packer.Ui, hook packer.Hook) (pack
 			HTTPAddress: b.config.HTTPAddress,
 		},
 		&vmwcommon.StepConfigureVNC{
-			Enabled:            !b.config.DisableVNC,
+			Enabled:            !b.config.DisableVNC && !b.config.VNCOverWebsocket,
```

Comment:
Nit: The double negation is a bit hard to process. Since DisableVNC is not really relevant when using VNCOverWebsocket I think we can do this

```suggestion
			Enabled:            !b.config.DisableVNC  || b.config.VNCOverWebsocket,
```

Reply:
This way `Enabled` will be enabled when `VNCOverWebsocket` is also enabled and this is not supposed to happen. To configure the VNC at this point both `DisableVNC` and `VNCOverWebsocket` should be `false`, because this one configures the common VNC connection, without websocket. 
DisableVNC is used to check whether to type the boot_command or not through VNC, so it should be false when using VNCoverWebsocket too.